styles.css

```diff
@@ -407,6 +456,231 @@
   line-height: 28px;
   color: #42526e;
   margin-top: 17px;
-  margin-top: 9px;
   margin-bottom: 40px;
 }
+
+/* media queries */
+
+@media (min-width: 576px) {
+  .toolbar {
+    padding-left: 50px;
+    padding-right: 50px;
+    padding-top: 26px;
+  }
+
+  .skills li {
+    font-weight: normal;
+    font-size: 14.1935px;
+    line-height: 28px;
+    font-style: normal;
+  }
+
+  .navbar .bar-icon {
+    display: none;
+  }
+
+  .navbar {
+    width: 50%;
+  }
+
+  .navbar ul {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .navbar ul li {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .headline-section {
+    margin-top: 140px;
+    line-height: 109px;
+    background: url("./assets/images/ellipse-desktop1.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
+  .headline-section h1 {
+    font-size: 75px;
+    margin-left: 142px;
+    margin-right: 142px;
+    line-height: 105px;
+  }
+
+  .headline-section h2 {
+    font-size: 40px;
+    line-height: 47px;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .headline-section p {
+    width: 600px;
+    font-size: 20px;
+    line-height: 32px;
+    font-weight: normal;
+  }
+
+  .about-section {
+    grid-template-columns: auto auto auto;
+    justify-content: center;
+    align-items: center;
+    background: url("./assets/images/ellipse-desktop1.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .title {
+    font-size: 18px;
+    line-height: 29px;
+    font-weight: bold;
+  }
+
+  .work-section-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-left: 90px;
+    margin-right: 90px;
+  }
+
+  .work-section-title-container1 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #a2a3ae;
+  }
+
+  .work-section-title-container2 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .social-icons ul {
+    align-items: center;
+  }
+
+  .recent-work {
+    font-size: 40px;
+    line-height: 114px;
+    font-weight: 700;
+  }
+
+  .contact {
+    display: none;
+  }
+
+  .chat {
+    display: flex;
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 32px;
+    line-height: 24px;
+    text-align: center;
+    color: #9390ff;
+    margin-bottom: 24px;
+    margin-top: 120px;
+  }
+
+  .contact-container p {
+    width: 400px;
+  }
+
+  .input {
+    width: 400px;
+  }
+
+  .client-message {
+    width: 400px;
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
- [ ]  In the main requirements of this project you are asked to use a breakpoint for your desktop version in the range of `992px - 1100px` as you can see in the screenshot below. Kindly please work on that

![req](https://user-images.githubusercontent.com/80895497/149553189-c68a4cd9-5ade-4ada-b8c0-573e9318ff0f.PNG)